ocaml/tests/test_xapi_xenops.ml

```diff
@@ -87,7 +87,6 @@ let test_xapi_restart_inner () =
     Xapi_xenops.start ~__context ~self:vm5 false false ;
     Xapi_xenops.start ~__context ~self:vm6 false false ;
     Xapi_xenops.start ~__context ~self:vm7 false false ;
-    (* vm6 is a ntnx CVM *)
     Db.VM.set_is_control_domain ~__context ~self:vm6 ~value:true ;
```

Comment:
Shouldn't this line be removed as well?

Reply:
I think the comment is not correct. The `vm6` is actually not a CVM. Its name doesn't end with "-CVM".